modules/FindFFTW3.cmake

```diff
@@ -23,41 +23,155 @@ if(FFTW3_INCLUDE_DIR AND FFTW3_LIBRARIES)
   return()
 endif()
 
-if(fftw3_FIND_COMPONENTS)
-    set(FFTW3_FIND_COMPONENTS ${fftw3_FIND_COMPONENTS})
-endif()
+macro(find_specific_libraries KIND PARALLEL)
+  list(APPEND FFTW3_FIND_COMPONENTS ${KIND}_${PARALLEL})
+  message(STATUS "========= ${${PARALLEL}_FOUND} ===========")
+  if(NOT (${PARALLEL} STREQUAL "SERIAL") AND NOT ${PARALLEL}_FOUND)
+    message(FATAL_ERROR "Please, find ${PARALLEL} libraries before FFTW")
+  endif()
+
+  message(STATUS "Checking for ${KIND} and ${PARALLEL}")
```

Comment:
Try and use `<package>_FIND_QUIETLY` rather than  always output something.
We shouldn't leave debug message behind.